### ISSUE TITLE
space-in-brackets: different rules for arrays and objects?

### ISSUE THREAD

**Issue by UltCombo** [eslint/eslint#2055]
It would be nice to have separate `space-in-brackets` rules for arrays and objects, to allow the following patterns to pass:

```js
var arr = [1, 2];
var obj = { a: 1, b: 2 };
```

And the following patterns to fail:

```js
var arr = [ 1, 2 ];
var obj = {a: 1, b: 2};
```

**Comment by UltCombo** [comment]
oops, pressed ctrl+Enter too early. Fixed OP.

**Comment by ilyavolodin** [comment]
Looks like a duplicate of #1576

**Comment by UltCombo** [comment]
That's true, thanks @ilyavolodin.
